guides/python-testing.md

```diff
@@ -100,25 +100,18 @@ push events (e.g., commits to GitHub) or to run in a preset cadence
 
 For GitHub, here are some services available:
 
+* GitHub Actions: This is a paid service billed centrally to STScI.
+  Repositories on ``spacetelescope`` GitHub organization should use it,
+  if possible.
 * Jenkins CI: This is an in-house CI solution with virtually no limitation
   (but do not push your luck). It supports storing big data and artifacts in
   Artifactory. Its internal-only instance also supports direct access to
   Central Storage. Unfortunately, it is not easily accessible for non-STScI
   personnels. It currently only supports Linux operating system. For more
```

Comment:
MacOS is available but your mileage may vary due to an extreme lack of resources (i.e there's one box).

The machine can be accessed using the labels `macos`, or `osx`.